src/services/getUniquePacklists.js

```diff
@@ -0,0 +1,9 @@
+export function getUniquePacklists(packlists) {
```

Comment:
I don't really understand why this is called an unique packlist, the naming is a bit confusing to me. Is Packlist already taken?

Reply:
these are the unique packlists, everyone can add a packlist, so sometimes, they are doubled :D